ui/raidboss/data/06-ew/raid/p6s.txt

```diff
@@ -1,7 +1,121 @@
 ### P6N: Abyssos: The Sixth Circle (Savage)
-#
+# -ii 7865 786A 7870 7871 7872 7873 7875 7879 787D 787E 7885 7886 7888 7889 7890 7892 79EB 79EC 7A18
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+9.5 "--sync--" sync / 14:[^:]*:Hegemone:7860:/ window 10,5
+14.2 "Hemitheos's Dark IV" sync / 1[56]:[^:]*:Hegemone:7860:/
+28.0 "Synergy" sync / 1[56]:[^:]*:Hegemone:(7887|788A):/
+35.1 "--sync--" sync / 1[56]:[^:]*:Hegemone:784C:/
+42.0 "Aetheric Polyominoid" sync / 1[56]:[^:]*:Hegemone:7866:/ window 15,15
+42.0 "--sync--" sync / 1[56]:[^:]*:Hegemone:7854:/
+48.6 "Unholy Darkness" sync / 1[56]:[^:]*:Hegemone:(786D|786E|7891):/
+54.6 "Polyominous Dark IV" sync / 1[56]:[^:]*:Hegemone:7867:/
+54.7 "Exocleaver 1" sync / 1[56]:[^:]*:Hegemone:(7869|786B):/ window 2,2
+56.8 "Unholy Darkness" sync / 1[56]:[^:]*:Hegemone:7892:/
+57.2 "Exocleaver 2" sync / 1[56]:[^:]*:Hegemone:(7869|786B):/ window 2,2
+71.2 "Pathogenic Cells x8" sync / 1[56]:[^:]*:Hegemone:7864:/ duration 15
+91.9 "--sync--" sync / 1[56]:[^:]*:Hegemone:784C:/
+97.7 "Aetherial Exchange" sync / 1[56]:[^:]*:Hegemone:784D:/ window 100,30
+104.4 "Exchange Of Agonies" sync / 1[56]:[^:]*:Hegemone:786C:/
+111.6 "Choros Ixou 1" sync / 1[56]:[^:]*:Hegemone:(7881|7882|7883|7884):/
+112.3 "Unholy Darkness" sync / 1[56]:[^:]*:Hegemone:(786D|786E|7891):/
+114.7 "Choros Ixou 2" sync / 1[56]:[^:]*:Hegemone:(7881|7882|7883|7884):/
+128.5 "Synergy" sync / 1[56]:[^:]*:Hegemone:(7887|788A):/
+141.4 "Hemitheos's Dark IV" sync / 1[56]:[^:]*:Hegemone:7860:/ window 30,30
+153.2 "Transmission" sync / 1[56]:[^:]*:Hegemone:7861:/
+159.0 "--sync--" sync / 1[56]:[^:]*:Hegemone:784C:/
+166.1 "Choros Ixou 1" sync / 1[56]:[^:]*:Hegemone:(7881|7882|7883|7884):/
+168.0 "Chelic Claw/Reek Havoc" sync / 1[56]:[^:]*:Parasitos:(79EE|79EF):/
+169.2 "Choros Ixou 2" sync / 1[56]:[^:]*:Hegemone:(7881|7882|7883|7884):/
+179.0 "Aetherial Exchange" sync / 1[56]:[^:]*:Hegemone:784D:/
+185.7 "Polyominoid Sigma" sync / 1[56]:[^:]*:Hegemone:7868:/ window 30,30
+185.7 "--sync--" sync / 1[56]:[^:]*:Hegemone:7854:/
+195.3 "Dark Dome 1" sync / 1[56]:[^:]*:Hegemone:788B:/
```

Comment:
For consistency, can you do the same thing with all 3 `Dark Dome`? The first ability is a castbar that leaves baited circles on the floor and the second ability is when those baited circles explode.  It's not really the same thing twice, so 1 and 2 numbering doesn't quite make sense here.

Maybe the second one should be `--circles--` or `--dark dome--` or something along those lines?